packages/core/src/actions/prepareTransactionRequest.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  type PrepareTransactionRequestErrorType as viem_PrepareTransactionRequestErrorType,
+  type PrepareTransactionRequestParameters as viem_PrepareTransactionRequestParameters,
+  type PrepareTransactionRequestReturnType as viem_PrepareTransactionRequestReturnType,
+  prepareTransactionRequest as viem_prepareTransactionRequest,
+} from 'viem/actions'
+
+import { type Config } from '../createConfig.js'
+import { type ChainIdParameter } from '../types/properties.js'
+import { type Evaluate, type UnionLooseOmit } from '../types/utils.js'
+
+export type PrepareTransactionRequestParameters<
+  config extends Config = Config,
+> = Evaluate<
+  UnionLooseOmit<viem_PrepareTransactionRequestParameters, 'chain'> &
+    ChainIdParameter<config>
+>
+
+export type PrepareTransactionRequestReturnType =
+  Evaluate<viem_PrepareTransactionRequestReturnType>
```

Comment:
Few generics to wire up here as well: https://github.com/wevm/viem/blob/main/src/actions/wallet/prepareTransactionRequest.ts#L76-L84